easybuild/easyconfigs/x/XZ/XZ-5.2.2-intel-2016a.eb

```diff
@@ -13,9 +13,6 @@ source_urls = ['http://tukaani.org/xz/']
 
 builddependencies = [
     ('Autotools', '20150215'),
-]
-
-dependencies = [
```

Comment:
then this PR cannot be merged until 2016b. It's one big versionsuffix conflict without this.